packages/microsoft-windows-terminal.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Microsoft Windows Terminal'
+$shortcutDir = ${Env:RAW_TOOLS_DIR}
+$shortcut = Join-Path $shortcutDir "$toolName.lnk"
+
+chocolatey uninstall "microsoft-windows-terminal.vm" --removedependencies
```

Comment:
I think we should remove this line because:
- It is not consistent with other packages
- It would give the user more flexibility. The user should use `chocolatey uninstall "microsoft-windows-terminal.vm" --removedependencies` if they want to remove `microsoft-windows-terminal`  completely and `chocolatey uninstall "microsoft-windows-terminal.vm"` if they want to keep microsoft-windows-terminal but not the tool in the Tools directory.
- Uninstalling is best effort so simpler code is better

Reply:
Sounds good, I'll change it!

There is one other package that does this, which is what left me unsure 😅
https://github.com/mandiant/VM-Packages/blob/fb843915dd35290aafef62c826dd43a574a0acaf/packages/visualstudio.vm/tools/chocolateyuninstall.ps1#L9